### PR TITLE
create default sections when creating a new discussion

### PR DESCRIPTION
**Pull request by vincentfretin**

### CODE REVIEW

File: assembl/models/discussion.py [hunk 1]
```diff
@@ -1086,6 +1087,66 @@ def date_to_piwik_date(date):
             }
 
 
+def create_default_sections(discussion):
```
[the remark this answers is not in the document]
Comment (vincentfretin) [reply assembl/models/discussion.py]: I can move it to `lib/migration.py`. In this case, maybe I should rename it to `create_default_discussion_data` and `create_default_discussion_sections`?

**Comment by vincentfretin** [comment]
I don't understand what you are talking about with the hook that starts pshell...
I did move the function to lib/migration.py though.

File: assembl/models/discussion.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ayyazdaniaryan) [reply assembl/models/discussion.py]: `create_default_discussion_data` would be cool 👍

**Comment by ayyazdaniaryan** [comment]
@vincentfretin You know when you load pshell local.ini, it loads all the locales and the permissions? I am not certain where that hook is in the code. Perhaps in the scripts/pshell.py? Or maybe it's part of the `__main__` in the pyramid's `__init__.py` file. I am not sure. Is that more clear?

**Comment by vincentfretin** [comment]
What? pshell just open a connection to the db, it doesn't load/create anything AFAICT.

**Comment by ayyazdaniaryan** [comment]
```
The WSGI application that is loaded will be available in the shell as the app global. Also, if the application that is loaded is the Pyramid app with no surrounding middleware, the root object returned by the default root factory, registry, and request will be available.```
https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/commandline.html

Okay so it's the pyramid app that is started. So, perhaps you want to include this logic of default sections in the `assembl/__init__.py#main`.

Somewhere in this transaction:
https://github.com/assembl/assembl/blob/develop/assembl/__init__.py#L90-L92

**Pull request merged**